### PR TITLE
Description Updates

### PR DESCRIPTION
**Pull request by sgtmind**
## About The Pull Request

Updates the descriptions for the obelisk and autolathes to give more detail. 

## Why It's Good For The Game
Not much description was given the obelisks which explained WHAT they did. I feel more information on what they do and their benefit would help compel NT to make more, creating a potential avenue for conflict. I also updated the autolathe description to give more flavor than the old description 

## Testing

Entered game, spawned items, checked items, all of them had updated descriptions. 

## Changelog
:cl:
tweak: Tweaked the descriptions of the autolathe, the excelsior autoforge, and the NT obelisk to give more description for what they do.

### CODE REVIEW

File: code/game/machinery/autolathe/autolathe.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/autolathe
 	name = "autolathe"
-	desc = "It produces items using metal and glass."
+	desc = "A general purpose fabricator capable of producing nearly any item you need, provided you have the necessary materials and design disk."
```
Comment (TheShown911) [inline code/game/machinery/autolathe/autolathe.dm:3]:
```suggestion
	desc = "A general purpose fabricator capable of producing nearly any item you need, provided you have the necessary materials and design disks."
```

File: code/game/machinery/excelsior/autolathe.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /obj/machinery/autolathe/excelsior
 	name = "excelsior autoforge"
-	desc = "It produces items using metal and glass."
+	desc = "A general purpose fabricator capable of producing nearly any item you may need for the revolution, provided you have the necessary materials and design disk"
```
Comment (TheShown911) [inline code/game/machinery/excelsior/autolathe.dm:3]:
```suggestion
	desc = "A general purpose fabricator capable of producing nearly any item you may need for the revolution, provided you have the necessary materials and design disks"
```

**Review by TheShown911 — APPROVED** [review]

File: code/modules/core_implant/cruciform/machinery/obelisk.dm [hunk 3]
```diff
@@ -2,7 +2,9 @@ GLOBAL_LIST_EMPTY(all_obelisk)
 
 /obj/machinery/power/nt_obelisk
 	name = "NeoTheology's obelisk"
-	desc = "The obelisk."
+	desc = "An obelisk of the Church which grants protection and strength to those who bear cruciforms."
+	description_info = "Obelisks help provide observation for the Eye, alongside granting faster sanity regeneration to NT faithful. In addition, when \"active\", they will repel weeds and damage hostile fauna such as roaches within range."
```
Comment (Humonitarian) [inline code/modules/core_implant/cruciform/machinery/obelisk.dm:6]: Do not use abbreviations in descriptions

**Review by Humonitarian — CHANGES_REQUESTED** [review]

File: code/modules/core_implant/cruciform/machinery/obelisk.dm (hunk 3 above)
Comment (sgtmind) [reply code/modules/core_implant/cruciform/machinery/obelisk.dm]: Fixed it

**Review by Humonitarian — APPROVED** [review]

**Pull request merged**